coinjoins/README.md

```diff
@@ -0,0 +1,79 @@
+# Coinjoins & Privacy UX Primer
+
+- Bitcoin is not anonymous.
+- Consider each address you generate as a new identity.
+- Coin has history
+- Histories can become linked
+- Public ledger
+- Anyone can lookup
+- New users may not be aware
+
+## Address Reuse
+
+Use a [hierarchical deterministic wallet](https://en.bitcoin.it/wiki/Deterministic_wallet) in order to avoid reusing an identity.
+
+Practicality
+
+- How do you adverise an address to be paid in a video?
+
+## Wallet Fingerprinting
+
+The way the transaction is constructed can give chain analysts an idea of which wallet was used.
+
+1. Change is sometimes the last output
+   1. Solutions
+      1. Lexicographical Ordering
+2. If the timelock in the transaction is a specific amount
+
+## Routing Transaction
+
+- Should use Tor
+  - Research Dandelion
+- If an attacker has enough nodes in the network it can determine the location of where the transaction was broadcasted.
+
+## Methods
+
+### Coinswap
+
+Privacy through a swap for another users coin history.
+
+### Coinjoins
+
+Privacy by combining of coin history.
+
+#### Censoring
+
+- Some exchanges may not accept deposits from coinjoins
+- Adding hops — *see Samourai Wallet's Ricochet*
+
+#### Components
+
+##### Rounds
+
+- Role of the coordinatior
```

Comment:
JoinMarket doesn't have a coordinator (technically the taker does the same job as the coordinator, i.e. putting together a transaction, but the privacy model is different)

perhaps a better typology is centralized coinjoins, using blind signatures like Samourai and wasabi, and peer to peer as in JoinMarket. I'm not aware of other implementations being in use.